.eslintrc.js

```diff
@@ -0,0 +1,98 @@
+module.exports = {
+  env: {
+    es6: true,
+    jest: true,
+    node: true,
+  },
+  extends: [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:@typescript-eslint/recommended-requiring-type-checking",
+    "prettier/@typescript-eslint",
+    "prettier",
+  ],
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    project: ["tsconfig.json"],
```

Comment:
Just FYI if you run into this problem later: when you have any JS files in the root of your project (e.g. config files), you will need `tsconfig.eslint.json` here (as in api.serlo.org).